Ambassador chapter [READY FOR REVIEW] (#3721)
the-turing-way/the-turing-way: book/website/collaboration/ambassador-schemes.md

(cl-ambassador-schemes)=
# Ambassador Schemes

(cl-ambassador-schemes-prerequisites)=

> # Prerequisites

Reviewer: ```suggestion
## Prerequisites
```

Reviewer: The reason for the missing subchapters was that this heading was on the wrong level.

| Prerequisites | Importance | Skill Level | Notes | 
| -------- | -------- | -------- |-------- |

> | {ref}`Collaboration <cl-newcommunity>`  | Helpful    | Beginner     | This chapter has lots of useful tips around managing a new community which can apply to ambassador schemes 

Author: ```suggestion
| {ref}`Collaboration <cl-new-community>`  | Helpful    | Beginner     | This chapter has lots of useful tips around managing a new community which can apply to ambassador schemes 
```

> | {ref}`Collaboration <cl-new-community>`  | Helpful    | Beginner     | This chapter has lots of useful tips around managing a new community which can apply to ambassador schemes 

Author: ```suggestion
| {ref}`Managing a New Community and Team <cl-new-community>`  | Helpful    | Beginner     | This chapter has lots of useful tips around managing a new community which can apply to ambassador schemes 
```

> | {ref}`Collaboration <cl-coworking>` and {ref}`Collaboration <cl-chairing>`  | Helpful    | Beginner     | These chapters may be useful if you are planning to host events and/or co-working during ambassador schemes

Author: ```suggestion
| {ref}`Organising Coworking Calls or Meetings <cl-coworking>` and {ref}`Chairing Events <cl-chairing>`  | Helpful    | Beginner     | These chapters may be useful if you are planning to host events and/or co-working during ambassador schemes
```

> | {ref}`Collaboration <cl-event-tools>`  | Helpful    | Beginner     | This chapter gives a useful overview of collaboration tools which can be deployed in an ambassador scheme

Author: ```suggestion
| {ref}`Tools for Collaboration <cl-event-tools>`  | Helpful    | Beginner     | This chapter gives a useful overview of collaboration tools which can be deployed in an ambassador scheme
```
